scripts/linkeySetup.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs');
+
+const dir = 'src/app/lib/config/services';
+const excludes = [
+  'archive',
+  'cymrufyw',
+  'loadableConfig',
+  'naidheachdan',
+  'scotland',
+];
+
+fs.readdir(dir, (err, files) => {
+  if (err) {
+    return err;
+  }
+
+  return files.forEach(configFileName => {
+    const fileName = configFileName.split('.');
+
+    const importConfig = `import { service } from './${fileName[0]}';\n`;
+    const testFile = fs.readFileSync('scripts/linkeyTest.js', (error, data) => {
+      return data;
+    });
+
+    if (excludes.indexOf(fileName[0]) >= 0) {
+      return null;
+    }
+
+    return fs.writeFile(
+      `${dir}/${fileName[0]}.test.js`,
+      importConfig + testFile,
+      'utf8',
+      error => {
+        if (error) throw error;
+      },
+    );
```

Comment:
I think that renaming some of the constants like this makes it more understandable,  but feel free to leave it as it is 🙂

Also, we can check if the service is one of the exclusive ones in the beginning.

```suggestion
    const serviceName = configFileName.split('.')[0];

    if (excludes.indexOf(serviceName) >= 0) {
      return null;
    }
    
    const importConfig = `import { service } from './${serviceName}';\n`;
    const testTemplate = fs.readFileSync(
      'scripts/linkeyTest.js',
      (error, data) => {
        return data;
      },
    );

    const serviceTestFile = `${dir}/${serviceName}.test.js`;
    return fs.writeFile(
      serviceTestFile,
      importConfig + testTemplate,
      'utf8',
      error => {
        if (error) throw error;
      },
    );
```